src/bandit-evaluator.ts

```diff
@@ -23,6 +23,45 @@ export class BanditEvaluator {
   private readonly assignmentShards = BANDIT_ASSIGNMENT_SHARDS; // We just hard code this for now
   private readonly sharder: Sharder = new MD5Sharder();
 
+  public evaluateBestBandit(
+    flagKey: string,
+    subjectKey: string,
+    subjectAttributes: ContextAttributes,
+    actions: Record<string, ContextAttributes>,
+    banditModel: BanditModelData,
+  ): BanditEvaluation {
+    const actionScores: Record<string, number> = this.scoreActions(
+      subjectAttributes,
+      actions,
+      banditModel,
+    );
+
+    const actionWeights: Record<string, number> = this.weighActions(
+      actionScores,
+      banditModel.gamma,
+      banditModel.actionProbabilityFloor,
+    );
+
+    const selectedActionKey = Object.keys(actionScores).reduce((prev, cur) => {
+      if (!prev || actionScores[cur] > actionScores[prev]) {
+        return cur;
+      }
+      return prev;
+    });
+
+    return {
+      flagKey,
+      subjectKey,
+      subjectAttributes,
+      actionKey: selectedActionKey,
+      actionAttributes: actions[selectedActionKey],
+      actionScore: actionScores[selectedActionKey],
+      actionWeight: actionWeights[selectedActionKey],
+      gamma: banditModel.gamma,
```

Comment:
I wonder if making this positive infinity would break anything, as selecting the best action is essentially the same as if gamma was positive infinity. 

In fact, that may be a way another way to achieve this 🤔 

Although this could complicate sending to data warehouses as they use different nomenclature than JavaScript. 🤔 But we aren't logging! 

My gut says to hard code this at `Infinity`

Reply:
going the simpler route here.